server/market/order/order_test.go

```diff
@@ -0,0 +1,389 @@
+// Package order defines the Order and Match types used throughout the DEX.
+package order
+
+import (
+	"encoding/hex"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/decred/dcrdex/server/account"
+	"github.com/decred/dcrdex/server/asset"
+)
+
+// func randomAccount() (acct account.AccountID) {
+// 	if _, err := rand.Read(acct[:]); err != nil {
+// 		panic("boom")
+// 	}
+// 	return
+// }
+
+var acct0 = account.AccountID{
+	0x22, 0x4c, 0xba, 0xaa, 0xfa, 0x80, 0xbf, 0x3b, 0xd1, 0xff, 0x73, 0x15,
+	0x90, 0xbc, 0xbd, 0xda, 0x5a, 0x76, 0xf9, 0x1e, 0x60, 0xa1, 0x56, 0x99,
+	0x46, 0x34, 0xe9, 0x1c, 0xec, 0x25, 0xd5, 0x40,
+}
+
+func TestPrefix_Serialize(t *testing.T) {
+	type fields struct {
+		AccountID  account.AccountID
+		BaseAsset  asset.AssetType
+		QuoteAsset asset.AssetType
+		OrderType  OrderType
+		ClientTime time.Time
+		ServerTime time.Time
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   []byte
+	}{
+		{
+			"ok acct0",
+			fields{
+				AccountID:  acct0,
+				BaseAsset:  asset.AssetDCR,
+				QuoteAsset: asset.AssetBTC,
+				OrderType:  LimitOrderType,
+				ClientTime: time.Unix(1566497653, 0),
+				ServerTime: time.Unix(1566497656, 0),
+			},
+			[]byte{0x22, 0x4c, 0xba, 0xaa, 0xfa, 0x80, 0xbf, 0x3b, 0xd1, 0xff, 0x73,
+				0x15, 0x90, 0xbc, 0xbd, 0xda, 0x5a, 0x76, 0xf9, 0x1e, 0x60, 0xa1,
+				0x56, 0x99, 0x46, 0x34, 0xe9, 0x1c, 0xec, 0x25, 0xd5, 0x40, 0x0,
+				0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x75, 0xdb, 0x5e,
+				0x5d, 0x0, 0x0, 0x0, 0x0, 0x78, 0xdb, 0x5e, 0x5d, 0x0, 0x0,
+				0x0, 0x0},
```

Comment:
Out of curiosity, how were these standards generated?

Reply:
Painfully.  :)  Doing `fmt.Println("%#v", field)` one field at a time and assembling the expected serialization.